src/shared/layout/GrowContainer.js

```diff
@@ -0,0 +1,53 @@
+import React, { useState } from 'react'
+import { Grid, Grow } from '@mui/material';
+import GridItem from './GridItem';
+import { Button } from 'shared/components/button/Button';
+
+const GrowContainer = ({ data, dataLength, upcoming, isMobile }) => {
+    const [grow, setGrow] = useState(false)
+    
+    const handleChange = () => {
+        setGrow(!grow);
+    };
+
+    return (
+        <>
+            {grow ? <>
+                <Grow in={grow}
+                    style={{ transformOrigin: '0 0 0', width: "100%" }}
+                    {...(grow ? { timeout: 3000 } : {})}
+                >
+                    <Grid item xs={1} sm={2} md={2}>
+                        <Grid container spacing={{ xs: 10, md: 12 }} columns={{ xs: 1, sm: 4, md: 6 }}>
+                            {data.slice(dataLength / 3, dataLength * 2 / 3).map((item, index) => (
+                                <GridItem item={item} key={index} />
```

Comment:
You forgot to put source into GridItem. If you press show more, the desc doesn't show (only pics).
![image](https://github.com/gisauweb/GISAUBC/assets/84741001/7a3b52bf-c887-4eee-a03d-b5cab21f12e5)

Reply:
Nice catch